rasa/model_training.py

```diff
@@ -199,10 +199,17 @@ def _train_graph(
 
     config = file_importer.get_config()
     recipe = Recipe.recipe_for_name(config.get("recipe"))
+    config = recipe.auto_configure(
```

Comment:
ok - so we only autoconfigure when training? I think that makes sense. Does this change the behaviour at all though? i.e. were there times we were autoconfiguring before but not now?

Reply:
Yes, only when we make use of recipes and only explicitly, by calling `auto_configure`. It used to be that importer would implicitly do this before. I don't see many other places using importers and everything seems fine with this change.